### PR TITLE
Update link_microsoft_low_reputation.yml

### PR DESCRIPTION
**Pull request by zoomequipd**
# Description

Add coverage for Microsoft image as content

# Associated samples

- [Sample 1](https://platform.sublime.security/messages/e9e47bf04b624be75508f7310ecb86b1872a524ec7b7354727d40a706195d229)

## Associated hunts

L14D (includes existing hits)
- [Hunt 1](https://platform.sublime.security/hunts/48e82ec1-31b0-4b58-8bf9-1f293bd91408)

### CODE REVIEW

**Comment by zoomequipd** [comment]
/mql-mimic-exempt 35416, 27744, 27673, 27669, 27680, 27657, 27577, 27578, 27573, 27463, 27550, 27555, 27556, 27580, 15242, 6158, 49172, 39062, 111343, 102638, 117919, 110269, 177806, 140579, 158720, 157123, 158742, 184836, 184887, 184885, 184898, 190746, 258637, 258670, 259484, 265980, 272554, 272557, 272556, 272555, 269044, 268469, 297154, 297158, 297181, 297182, 297258, 297270, 297442, 306512, 229064, 228149, 228308, 221366, 228259, 228323, 228307, 228250, 221352, 238433, 228329, 238423, 228180, 125583, 223566, 223571, 228324, 326363, 266038, 276223, 351979, 351980, 352303, 102908, 221353, 221360, 228258, 228305, 352571, 352577, 362854, 363216, 428277, 471067, 479210, 514509, 515967, 520366, 520367, 520368, 520785

exclude previously excluded messages

**Comment by zoomequipd** [comment]
/mql-mimic-exempt: 6158, 102638, 102908, 111343, 117919, 125583, 140579, 15242, 157123, 158720, 158742, 177806, 184836, 184885, 184887, 184898, 190746, 221352, 221353, 221360, 221366, 223566, 223571, 228149, 228180, 228250, 228258, 228259, 228305, 228307, 228308, 228323, 228324, 228329, 229064, 238423, 238433, 258637, 258670, 259484, 265980, 266038, 269044, 272554, 272555, 272556, 272557, 27657, 27669, 27673, 27680, 27550, 27555, 27556, 27573, 27577, 27578, 27580, 276223, 27744, 297154, 297158, 297181, 297182, 297258, 297270, 297442, 306512, 326363, 351979, 351980, 352303, 352571, 352577, 362854, 363216, 35416, 428277, 471067, 479210, 514509, 515967, 520366, 520367, 520368, 520785, 627271, 645729

 627271 and 645729 actually fire in analyzer, rest are previously excluded

File: detection-rules/link_microsoft_low_reputation.yml [hunk 1]
```diff
@@ -205,7 +206,18 @@ source: |
  
  // suspicious content
  and (
+   // current thread is empty, but an image attachment is braded as microsoft.
```
Comment (aidenmitchell) [inline detection-rules/link_microsoft_low_reputation.yml:209]:
```suggestion
   // current thread is empty, but an image attachment is branded as microsoft.
```

**Review by aidenmitchell — APPROVED** [review]

**Pull request merged**